From: Tejun Heo <teheo@suse.de>
Subject: sata_nv: disable ADMA by default
References: 346508

Disable ADMA by default.  There are several unresolved issues leading
to FLUSH timeouts, hard lockup during resume and timeouts.

Signed-off-by: Tejun Heo <teheo@suse.de>
---
 drivers/ata/sata_nv.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/ata/sata_nv.c
+++ b/drivers/ata/sata_nv.c
@@ -640,7 +640,7 @@ MODULE_LICENSE("GPL");
 MODULE_DEVICE_TABLE(pci, nv_pci_tbl);
 MODULE_VERSION(DRV_VERSION);
 
-static int adma_enabled = 1;
+static int adma_enabled = 0;
 static int swncq_enabled;
 
 static void nv_adma_register_mode(struct ata_port *ap)
